### PR TITLE
[STK-256] Go build and release

### PR DESCRIPTION
**Pull request by faizanahmad055**

### CODE REVIEW

**Comment by stakater-user** [comment]
@faizanahmad055 Image is available for testing. ``docker pull stakater/chowkidar:PR-25-1``

**Comment by stakater-user** [comment]
@faizanahmad055 Yikes! You better fix it before anyone else finds out! [Build 4](https://jenkins.tools.stackator.com/job/Stakater/job/Chowkidar/job/PR-25/4/) has Failed!

**Comment by stakater-user** [comment]
@faizanahmad055 Yikes! You better fix it before anyone else finds out! [Build 5](https://jenkins.tools.stackator.com/job/Stakater/job/Chowkidar/job/PR-25/5/) has Failed!

**Comment by stakater-user** [comment]
@faizanahmad055 Yikes! You better fix it before anyone else finds out! [Build 6](https://jenkins.tools.stackator.com/job/Stakater/job/Chowkidar/job/PR-25/6/) has Failed!

**Comment by stakater-user** [comment]
@faizanahmad055 Yikes! You better fix it before anyone else finds out! [Build 7](https://jenkins.tools.stackator.com/job/Stakater/job/Chowkidar/job/PR-25/7/) has Failed!

**Comment by stakater-user** [comment]
@faizanahmad055 Yikes! You better fix it before anyone else finds out! [Build 8](https://jenkins.tools.stackator.com/job/Stakater/job/Chowkidar/job/PR-25/8/) has Failed!

**Comment by stakater-user** [comment]
@faizanahmad055 Yikes! You better fix it before anyone else finds out! [Build 9](https://jenkins.tools.stackator.com/job/Stakater/job/Chowkidar/job/PR-25/9/) has Failed!

**Comment by stakater-user** [comment]
@faizanahmad055 Image is available for testing. ``docker pull stakater/chowkidar:PR-25-10``

File: Makefile [hunk 1]
```diff
@@ -19,19 +19,19 @@ LDFLAGS =
 
 default: build test
 
-install: 
+install:
 	"$(GLIDECMD)" install
-	cp -r vendor/* ${GOPATH}/src/ 
+	cp -r vendor/* ${GOPATH}/src/
```
Comment (waseem-h) [inline Makefile]: Don't copy vendor folder as it is not necessary

**Comment by stakater-user** [comment]
@faizanahmad055 Yikes! You better fix it before anyone else finds out! [Build 12](https://jenkins.tools.stackator.com/job/Stakater/job/Chowkidar/job/PR-25/12/) has Failed!

**Comment by stakater-user** [comment]
@faizanahmad055 Image is available for testing. ``docker pull stakater/chowkidar:PR-25-13``

**Comment by stakater-user** [comment]
@faizanahmad055 Image is available for testing. ``docker pull stakater/chowkidar:PR-25-14``

File: Jenkinsfile [hunk 2]
```diff
@@ -1,144 +1,8 @@
 #!/usr/bin/groovy
-@Library('github.com/stakater/fabric8-pipeline-library@master')
+@Library('github.com/stakater/fabric8-pipeline-library@goBuildAndRelease')
```
Comment (hazim1093) [inline Jenkinsfile]: Use tag v2.4.0 instead of branch or master

**Comment by stakater-user** [comment]
@faizanahmad055 Image is available for testing. ``docker pull stakater/chowkidar:PR-25-15``

**Pull request merged**